### DISCUSSION TITLE
Issues가 readme에 자동반영될려면

### DISCUSSION THREAD

**Discussion by fkdl0048** [fkdl0048/Mentoring-Project#14] [Q&A] [answered]
issues에서 close되거나 add하게 되면 readme에 자동반영되게할려면 github Action을 쓰는방법밖에 없을까요?

좀 더 자동화를 만들고 싶은데 검색해보니 파이썬을 사용해서 Actions에 설정하면 커밋은 자동 반영이 된다고 해서 궁금합니다.!

**Comment by jongfeel** [comment, marked as answer]
일관된 규칙이 있으면 좋긴 하겠지만
(issue 제목 그대로 readme 파일 마지막줄에 추가한다던가 하는 규칙)

결국 Readme 파일도 format 이라는게 있으니까 원하는 대로 작성하려면 자동화는 어려울 수 있긴 합니다.

저의 경우에는 좀 귀찮지만 수동으로 이렇게 합니다.

- issue로 스케줄 항목 생성 (멘토링, 미팅 참석 등)
- issue 작업이 완료가 되면 issue의 branch 만드는 기능을 활용해서 branch 생성
- 해당 branch를 로컬에 fetch 한 후에 readme 파일 수정
- readme 파일 commit, push 후 pull reqeust 생성 (자동으로 issue와 연결됨)
- Auto and squash merge, delete branch로 마무리

![image](https://user-images.githubusercontent.com/17442457/197671250-e6c8610e-bd10-4429-8cda-5d155a02e526.png)

자동화의 목적은 없는 프로세스이지만
작업의 기록을 남기는 방식으로는 괜찮은 프로세스라서 하고 있어요.
